edx/cookiecutter-django-app
# Improvements from enterprise app

**Description:** This PR contains fixes and improvements found working on new edx-enterprise app.

**JIRA:** [ENT-21](https://openedx.atlassian.net/browse/ENT-21)

**Testing instructions:**
- Run tests and make sure they are passing.
- Try generating new app (as described in this repo's README) and check the following:
  *\* pydocstyle does not require all classes to have docstrings (arguable decision, see https://github.com/edx/edx-enterprise/pull/1#discussion_r84101811 for motivation)
- generated docs/{{appname}}.rst is gitignored
- `make docs` return zero violations
- Django is added to dependency list in setup.py
- test_utils package is present (used to workaround "tests folder should not contain **init**.py file")
- `make quality` report zero violations

**Reviewers:**
- [x] @mtyaka (internal review at Opencraft)
- [x] @clintonb
- [ ] @edx/platform-team
